sycl/test/warnings/sycl_2020_deprecations.cpp

```diff
@@ -150,7 +150,7 @@ int main() {
   sycl::multi_ptr<int, sycl::access::address_space::global_space,
                   sycl::access::decorated::yes>
       a(nullptr);
-  // expected-warning@+1 {{'atomic<int, sycl::access::address_space::global_space>' is deprecated: sycl::atomic is deprecated since SYCL 2020}}
+  // expected-warning@+1 {{'atomic' is deprecated: sycl::atomic is deprecated since SYCL 2020}}
```

Comment:
The text after `'atomic' is deprecated:` should not repeat that the same phrase, but instead should say "use `sycl::atomic_ref` instead"